docs/usage/configuration-options.md

```diff
@@ -2363,8 +2365,11 @@ after 10pm and before 5:00am
 after 10pm and before 5am every weekday
 on friday and saturday
 every 3 months on the first day of the month
+* 0 2 * * *
```

Comment:
Are you sure the `cron` schedule example is correct? I think we need to drop the last `*` from the example, like this:

```suggestion
* 0 2 * *
```

My suggestion converts to: "At every minute past hour 0 on day-of-month 2." according to crontab guru [^guru].

[^guru]: https://crontab.guru/#*_0_2_*_*

Reply:
Hi @HonkingGoose oh, sorry, that was a wrong example.